src/main/kotlin/mate/academy/service/AriaCalculator.kt

```diff
@@ -0,0 +1,6 @@
+package mate.academy.service
+
+interface AriaCalculator {

```

Comment:
Mistype in naming
![Screenshot 2024-03-27 143606](https://github.com/mate-academy/kt-oop-figures/assets/83113910/7d7d221f-a70c-43b4-9ec0-3cd4a89c12e1)

Reply:
oh, thanks a lot